helm/cert-manager-app/values.yaml

```diff
@@ -235,3 +234,13 @@ webhook:
   # webhook.validatingWebhookConfigurationAnnotations
   # Optional additional annotations to add to the webhook ValidatingWebhookConfiguration
   validatingWebhookConfigurationAnnotations: {}
+
+Installation:
+  V1:
+    GiantSwarm:
+      CertManager:
+        AcmeSolver: http01
+
+    Secret:
+      Cloudflare:
+        Token: ""
```

Comment:
This structure implies that there need to be 2 values set properly for the thing to work. I think in this case we can reduce to something like:

```yaml
Installation:
  V1:
    GiantSwarm:
      CertManager:
        AcmeSolver:
          Http01:
            CloudflareToken: ""
```

Reply:
But that's how we split sensitive/non-sensitive data in other services.

Also I expect this to change soon after migration to config repo